### DISCUSSION TITLE
2022-24: 开源，无禁止即可为

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#360] [Announcements]
# 2022-24: 开源，无禁止即可为

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-24/

**Comment by gaocegege** [comment]
Thanks for the contributions to envd! 

实际上除了 @VoVAllen 和我是全职投入，其他的朋友都是被邀请进来一起开源协作的。

**Reply by Xuanwo** [reply]
原来如此，他们的贡献太多了，我还以为是全职的😂😂😂

**Reply by gaocegege** [reply]
只能说大家都喜欢摸！

**Reply by Triple-Z** [reply]
滋辞高策哥哥！

**Comment by tianheg** [comment]
写得不错，增强了我参与开源的决心。

一个错别字：
> 主**哟**团队成员包括

**Reply by Xuanwo** [reply]
感谢指正，已经修复啦
